### PR TITLE
Fix RZUSBStick issue where device is left in bad state after use by r…

### PR DESCRIPTION
**Pull request by Crypt0s**
…esetting the USB device each time it is instantiated.

### CODE REVIEW

**Comment by Crypt0s** [comment]
Incidentally, if you attempt to dev.reset() the connection before you do configure() on it, you will cause the pyUSB library to SEGFAULT -- this may be something we wish to file a bug on later.

File: tools/zbpanidconflictflood [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+import argparse
+from time import sleep
+import threading
+import logging
+logging.getLogger("scapy.runtime").setLevel(logging.ERROR) # suppress annoying Scapy IPv6 warning
+from killerbee import *
+from scapy.all import *
+
+# Jacked from killerbee project.
+def randmac(length=8):
```
Comment (riverloopsec) [inline tools/zbpanidconflictflood]: Any chance we could put randmac into the kbutils or some similar location in the main project so it doesn't have to live in multiple scripts?

File: tools/zbpanidconflictflood [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+    prefix = random.choice(prefixes)
+    suffix = randbytes(length-3)
+    # Reverse the address for use in a packet
+    return ''.join([prefix, suffix])[::-1]
+
+
+def create_beacon(panid, coordinator, epanid):
+    # oooh, baby I like it RAWWWWWWWWWWWW
+    print hex(panid),
```
Comment (riverloopsec) [inline tools/zbpanidconflictflood]: Line maybe left over from debugging?

File: tools/zbrealign [hunk 3]
```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python
```
Comment (riverloopsec) [inline tools/zbrealign]: Have you had success with realignment frames on commercial devices? We haven't had success with these in-the-wild in the past. Would be interested to learn more - perhaps discuss offline.

**Comment by Crypt0s** [comment]
Ohhhhh man, I did NOT mean to sub this pull request with anything except the commit for the reset issue -- lets close this one out while I prep.  I have a few discussion items re: project org. with you as well.  I will reach out over email.

**Pull request closed without merging**